index.html

```diff
@@ -38,5 +38,181 @@ <h1 class="lines"> Hey there <br>
           <img src="images/vector.svg" alt="normalbottun">
         </div>
     </section>
+    
+
+    <section class="line-break">
```

Comment:
- [ ]Kindly make sure to have only one section for my recent works. For clean code purposes